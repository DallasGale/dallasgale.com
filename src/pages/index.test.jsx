import React from 'react'
import renderer from 'react-test-renderer'
import IndexPage from '.'

describe('IndexPage', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<IndexPage />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
