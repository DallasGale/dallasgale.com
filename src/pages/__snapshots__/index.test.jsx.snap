// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
<button
  onClick={[Function]}
  type="button"
>
  Report Feedback
</button>
`;
